cime_config/machines/cmake_macros/nvidia_pm-cpu.cmake

```diff
@@ -17,7 +17,7 @@ if (NOT DEBUG)
   string(APPEND CXXFLAGS " -O2")
 endif()
 if (NOT DEBUG)
-  string(APPEND FFLAGS " -O2")
+  string(APPEND FFLAGS " -g")
```

Comment:
Adding -g for NOT DEBUG ?

Reply:
Yea, I didn't mention that in the comments -- I like `-g` for builds as it can sometimes give more info about a crash. It will increase exe size, but I don't see major performance impacts -- esp for situations like this where we are only using nvidia compiler on this machine as a test. But I can remove it and only add it when I'm testing.

Reply:
I was just wondering if its a mistake.  Its fine to leave it.